aca_preview/preview.py

```diff
@@ -471,8 +471,8 @@ def get_text_pre(self):
 Probability of acquiring 2 or fewer stars (10^-x): {P2:.2f}
 Acquisition Stars Expected: {self.acq_count:.2f}
 Guide Stars count: {self.guide_count:.2f}
-Predicted Guide CCD temperature (max): {self.t_ccd_guide:.1f}
-Predicted Acq CCD temperature (init) : {self.t_ccd_acq:.1f}"""
+Predicted Guide CCD temperature (max): {self.acqs.t_ccd:.1f}
```

Comment:
A little confused about this because I think you do want the guide temperature if you have it (to display what was actually used in proseco selection as debug output).

Reply:
Ooops.  c659648.